crates/runc-shim/Cargo.toml

```diff
@@ -40,6 +40,7 @@ uuid.workspace = true
 # Async dependencies
 async-trait.workspace = true
 tokio = { workspace = true, features = ["full"] }
+tokio-eventfd = "0.2.1"
```

Comment:
I guess this needs to be moved under linux specific dependencies to keep CI green.

Reply:
done